### PR TITLE
Step 13 Mochawesome

### PR DESCRIPTION
**Pull request by Antonieta13**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,8 +7,11 @@
     "test:open": "npm run lint && cypress open",
     "lint": "eslint ./cypress/integration/**/*.ts",
     "lint:fix": "npm run lint -- --fix",
-    "test": "cypress run"
+    "test": "cypress run",
+    "combine-reports": "mochawesome-merge cypress/reports/mocha/*.json > cypress/reports/report.json",
+  "generate-report": "marge cypress/reports/report.json -f report -o cypress/reports"
```
Comment (holgiosalos) [inline package.json]: fix the indentation

**Review by holgiosalos — COMMENTED** [review]
Great job! can you upload an screenshot of the report generated :)

File: package.json (hunk 1 above)
Comment (Antonieta13) [reply package.json]: Done

Comment (Antonieta13) [reply package.json]: I just finished the indentation, :)  I will work on the report and send the screenshot.

Comment (holgiosalos) [reply package.json]: Yeah, please remember to use the npm pre and post hooks to run scripts before and after tests

Comment (Antonieta13) [reply package.json]: ReporteMochawesomeAnto


Finally!

Comment (Antonieta13) [reply package.json]: I had some problems, the steps on de workshop say to install mochawesome npm install mocha --save-dev, but to me, worked whit npm install mochawesome --save-dev. 

I found the cypress documentation and that help me. 

https://docs.cypress.io/guides/tooling/reporters#Examples
https://www.yld.io/blog/using-npm-pre-and-post-hooks/

Also, the terminal (I worked with the windows terminal) doesn't recognize the RM command, which confused me a lot, but then I understood that command is for Linux. :S

**Review by holgiosalos — COMMENTED** [review]
Hi @Antonieta13 

please delete de mochawesome-report dir and the mochawesome.json file. And commit the deletion of those files.

File: package.json (hunk 1 above)
Comment (holgiosalos) [reply package.json]: Really good job anto!

Comment (Antonieta13) [reply package.json]: :) Thank you, and thank you to you for teaching us this. 

I am done with the changes!

File: cypress.json [hunk 2]
```diff
@@ -1,5 +1,17 @@
 {
     "video": false,
     "defaultCommandTimeout": 20000,
-    "responseTimeout": 20000
+    "responseTimeout": 20000,
+
```
Comment (renardete) [inline cypress.json:5]: delete unnecessary line please

Comment (Antonieta13) [reply cypress.json:5]: Done

**Review by renardete — APPROVED** [review]

**Pull request merged**